src/schema/comments.ts

```diff
@@ -954,13 +955,15 @@ export const resolvers: IResolvers<any, Context> = {
         const post = await comment.post;
         await ensureSourcePermissions(ctx, post.sourceId);
         await ctx.con.transaction(async (entityManager) => {
+          await entityManager.getRepository(UserComment).save({
+            commentId: id,
+            userId: ctx.userId,
+            vote: UserVote.Up,
+          });
           await entityManager.getRepository(CommentUpvote).insert({
```

Comment:
One of the two here can eventually also be cleaned correct? Maybe just add todo as well